dali/test/python/test_operator_coord_transform.py

```diff
@@ -125,45 +132,53 @@ def _run_test(device, batch_size, out_dim, in_dim, in_dtype, out_dtype, M_kind,
         if out_dtype != np.float32:  # headroom for rounding
             avg += 0.33
             eps += 0.5
-        check_batch(outputs[1], ref, batch_size, eps, eps, expected_layout="NX", compare_layouts=True)
+        check_batch(outputs[1], ref, batch_size, eps, eps, expected_layout="NX",
+                    compare_layouts=True)
```

Comment:
No need to repeat the default argument
```suggestion
        check_batch(outputs[1], ref, batch_size, eps, eps, expected_layout="NX")
```